pkg/pdatatest/plogtest/logs.go

```diff
@@ -174,16 +227,21 @@ func CompareLogRecordSlices(expected, actual plog.LogRecordSlice) error {
 	}
 
 	for alr, elr := range matchingLogRecords {
-		if err := CompareLogRecords(alr, elr); err != nil {
-			return multierr.Combine(fmt.Errorf("log record with attributes: %v, does not match expected", alr.Attributes().AsRaw()), err)
+		if err := CompareLogRecord(alr, elr); err != nil {
+			return multierr.Combine(fmt.Errorf("log record with attributes %v does not match expected", alr.Attributes().AsRaw()), err)
```

Comment:
```suggestion
			return multierr.Combine(fmt.Errorf("LogRecord with attributes %v does not match expected", alr.Attributes().AsRaw()), err)
```
This seems to match the casing used elsewhere.

Reply:
`log record` is actually being used across all the logs.go. I can update it in a separate PR that would polish all the errors